.github/workflows/nodejs.yml

```diff
@@ -7,15 +7,14 @@ on:
   push:
     branches: [master]
   pull_request:
-    branches: [master]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
good thinking. we should investigate adding windows to this list too if a VM is available. Can you log an issue in that direction?